pkg/cmd/pulumi/new.go

```diff
@@ -1083,6 +1090,21 @@ func templatesToOptionArrayAndMap(templates []workspace.Template,
 	return options, nameToTemplateMap
 }
 
+// formatDesc formats the template description nicely if the term width requires it to wrap around.
+func formatDesc(prefixWidth int, maxWidth int, desc string) string {
+	descFormatted := ""
+	descWidth := maxWidth - prefixWidth
+	for idx := descWidth; idx < len(desc); idx += descWidth {
+		buffer := prefixWidth
+		if strings.LastIndex(desc[:idx], " ") != -1 {
+			idx = strings.LastIndex(desc[:idx], " ")
```

Comment:
Minor thing: This will calculate the LastIndex twice.
We can use a `:=` in the if statement to fix that:
```suggestion
		if lastSpace := strings.LastIndex(desc[:idx], " "); lastSpace != -1 {
			idx = lastSpace
```